man/systemd.service.xml

```diff
@@ -897,6 +897,28 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestartMode=</varname></term>
+
+        <listitem>
+          <para>Takes a string value that specifies how a service should restart:
+            <itemizedlist>
+              <listitem><para>If set to <option>normal</option> (the default), the service restarts by
+              going through a failed/inactive state.</para></listitem>
+
+              <listitem><para>If set to <option>quick</option>, the service transitions to the activating
```

Comment:
bike shedding, but: `quick` seems awfully unclear to me, as this is not about speed, but skipping some steps. So how about `skip-notifications` or `skip-triggers` instead?

Reply:
`skip-inactive-state`? It's how it's described in this manpage - `quick` by itself really doesn't convey the right message, ie, the opposite wouldn't be `slow`

Reply:
i don't like skip-inactive-state, since it skips both the inactive state and the failed state.

Reply:
By popular acclamation (a telegram poll) we have settled on `direct` instead of `quick` - @rphibel could you please do the change? Then it's good to go